### PR TITLE
3 alpha sort pulldowns

### LINKED ISSUE #3: [1:1] Alphabetically sort pulldowns/select
Please ensure alphabetical sorting in all pulldowns that do not have an explicit sort already specified in the code base.

Acceptance Criterion:
- [x] A comment on indicating any pulldowns that you chose not to sort alphabetically along with a BRIEF justification for why you made the decision to not alphabetize

- [x] Application of alphabetical sorting to the remaining pulldowns.

- [x] Manual testing confirm that alphabetical sorting was properly implemented.

### PR DESCRIPTION
**Pull request by Randond**
Resolves #3 

### Description

This PR replaces #9, which was using the wrong branch. See review and comments there.

### Type of change

* Bug fix (non-breaking change which fixes an issue)
* New feature (non-breaking change which adds functionality)
* Breaking change (fix or feature that would cause existing functionality to not work as expected)
* This change requires a documentation update
* Documentation update

### CODE REVIEW

**Comment by Randond** [comment]
@mdworken I made the change removing `collect` as you suggested. Please review again and approve this if you think it is ready.

File: app/views/partner_requests/_item.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <tr>
-  <td><%= form.select :name, POSSIBLE_ITEMS.slice(*@valid_items).invert.collect, {include_blank: 'Select an item'}, {class: 'form-control'} %></td>
+<td><%= form.select :name, POSSIBLE_ITEMS.slice(*@valid_items).invert.sort_by{|k,v|k}, {include_blank: 'Select an item'}, {class: 'form-control'} %></td>
```
Comment (armahillo) [inline app/views/partner_requests/_item.html.erb:2]: Nice :100:

Comment (mdworken) [reply app/views/partner_requests/_item.html.erb:2]: Looks great! Did you boot the app with these changes and make sure it still looks right?

**Review by mdworken — APPROVED** [review]
Looks great!

File: app/views/partner_requests/_item.html.erb (hunk 1 above)
Comment (Randond) [reply app/views/partner_requests/_item.html.erb:2]: Yes.  It still looks good!

**Comment by Randond** [comment]
@mdworken I think that this is ready.  If so, can you merge?

**Pull request merged**